config.yml

```diff
@@ -27,6 +27,7 @@ default:
         oncodriveclustl:
                 clusters: "versioned_results/oncodriveclustl-0.0_{abase}_clusters_results.tsv"
                 elements: "versioned_results/oncodriveclustl-0.0_{abase}_elements_results.tsv"
+        curated_blacklist: "versioned_results/curated_blacklist/candidateblacklist.schmitz.{projection}.tsv"
```

Comment:
@hayashaalan is this file already in GAMBL? Just want to make sure it is on master so everyone has access to it and then this can be merged. Thanks!

Reply:
I have a GAMBL PR that contains the necessary files!